src/extension.ts

```diff
@@ -75,6 +86,41 @@ export function activate(context: vscode.ExtensionContext) {
 	let dartIndentFixer = new DartIndentFixer();
 	context.subscriptions.push(vscode.window.onDidChangeActiveTextEditor(td => dartIndentFixer.onDidChangeActiveTextEditor(td)));
 	dartIndentFixer.onDidChangeActiveTextEditor(vscode.window.activeTextEditor); // Handle already-open file.
+
+	context.subscriptions.push(vscode.workspace.onDidChangeConfiguration(handleConfigurationChange));
+}
+
+function handleServerStatus(status: ServerStatusNotification) {
```

Comment:
[Done this](https://github.com/DanTup/Dart-Code/commit/e3211ed86ed827679bb7263a78e620c20ae24afc); hope it seems reasonable!